### ISSUE TITLE
document float and int values for PyTime

### ISSUE THREAD

**Issue by RRoenn** [mhammond/pywin32#1670]
### Expected behavior and actual behavior.
_edit to match the title change:_ 
- expected behavior: int(PyTime) and float(PyTime) cast to same value with different precision, e.g.  36525.9583333 and 36525
- actual behavior: they cast to different values ==> int(PyTime) and float(PyTime) have different meanings, see below

### Steps to reproduce the problem.
```
from win32file import CreateFile, SetFileTime, GetFileTime, CloseHandle 
from win32file import GENERIC_READ, GENERIC_WRITE, OPEN_EXISTING, FILE_FLAG_BACKUP_SEMANTICS, FILE_SHARE_WRITE
from pywintypes import Time

  fh = CreateFile( path_to_file, GENERIC_READ | GENERIC_WRITE, 0, None, OPEN_EXISTING, 0, 0 )
  createTime, accessTime, modifyTime = win32file.GetFileTime( fh )
  print( createTime )          # e.g. 12/31/99 23:00:00
  print( int(createTime) )     # 946677600
  print( float(createTime) )   # 36525.9583333
```

### Version of Python and pywin32
python version 2.7.13.1
pywin32 version 228

**Comment by RoccoMatano** [comment]
I don't see any problem here. PyTime wraps a [DATE](https://docs.microsoft.com/en-us/cpp/atl-mfc-shared/date-type?view=msvc-160), which is nothing more than a 64 bit float.  36525.9583333 is exactly the value that is used to represent 12/31/99 23:00:00. FYI: Excel is using the same way to represent dates.

**Comment by RRoenn** [comment]
> __int__
>     Used when an integer representation of the time object is required. 
> 
> __float__
>     Used when a floating point representation of the time object is required.


I thought if 36525.9583333 is float then int is 36525

So what is my thoughtless error?
Or have int(date) and float(date) different meanings?

Thank you for the clarification.

**Comment by RoccoMatano** [comment]
>Or have int(date) and float(date) different meanings?

Exactly. While integer format counts the seconds since January 1, 1970, 00:00:00, the flot format counts the days since 30 December 1899 midnight. And of course the are different numbers.

**Comment by RRoenn** [comment]
Thank you.
I did't find that in the documentation or looked at the wrong place.

**Comment by mhammond** [comment]
> I did't find that in the documentation

Yeah, I don't think that is documented, so let's make this issue about that.

**Comment by artREGRESION** [comment]
hello @mhammond is there any way i can contact you about a question i have with python? I am doing the final project of my career and I do not know very well how to access the first sector of an image or device to obtain the partitions etc. Thank you and sorry if there is no place for this message

**Comment by mhammond** [comment]
> hello @mhammond is there any way i can contact you about a question i have with python

Please see https://github.com/mhammond/pywin32#support

**Comment by kxrob** [comment]
> Yeah, I don't think that is documented, so let's make this issue about that.
> .. closed this in 48d23ba


Note: it seems on Py3 `<PyDateTime>` objects are used throughout instead:
(A subclass of Pythons `<datetime.datetime>` with added .Format())

```
DBT>>> createTime, accessTime, modifyTime = win32file.GetFileTime(h)
DBT>>> createTime
pywintypes.datetime(2020, 7, 30, 8, 25, 34, 943000, tzinfo=TimeZoneInfo('GMT Standard Time', True))
DBT>>> createTime.timestamp()
1596097534.943
DBT>>> float(createTime)
Traceback (most recent call last):
  File "<interactive input>", line 1, in <module>
TypeError: float() argument must be a string or a number, not 'pywintypes.datetime'
```

=> GetFileTime etc. (a lot) documentation still points to PyTime.

(Guess its not necessary or reasonable to add `.__float__ / .__int__()` to PyDateTime as Py3 code was forced into that for long time, Py2 code conversion is anyway a big step, and its seems better to enforce the explicit methods and normal datetime compat., and the jump between float and int time scale was indeed somewhat surprising.)

**Comment by mhammond** [comment]
Yeah, I was a bit too quick here, and you are completely correct that my edit doesn't make sense given we've killed py2. Rather than try to clarify things, I just opened #1677 which is a bit more nuclear but should remove future confusion.
